sdk/cloudmachine/README.md

```diff
@@ -1,143 +1,93 @@
 # Azure Projects client library for .NET
 
-Write Azure apps in 5 minutes
+Azure.Projects is an set of libraries and tools for rapid application development on Azure.
+If you cannot get an app running in 10 minutes, let us know that we failed!
 
-## Getting started
-
-### Install the packages
+These libraries and tools make it easy to get apps off the ground, by:
+- relying on opinionated defaults whenever possible.
+- using the Azure.Provisioning for provisioning resources in code (in C#).
+- exposing simplified APIs for the most commonly used Azure services.
 
-Install the client library for .NET with [NuGet](https://www.nuget.org/ ):
+At the same time, Azure.Projects supports break-glass scenarios where, if needed, you can override the defaults, use powerful tools (like bicep, azd), or the full featured Azure SDK libraries. 
+In other words, Azure.Projects provides smart, simplified APIs but will never block you from using the full power of the Azure platform, if you choose to. 
 
-```dotnetcli
-dotnet add package Azure.Projects --prerelease
-dotnet add package Azure.Projects.Provisioning --prerelease
-dotnet add package Azure.Projects.AI --prerelease
-```
-
-### Authenticate the Client
+## Getting started
 
 ### Prerequisites
 
 * You must have an [Azure subscription](https://azure.microsoft.com/free/dotnet/).
-* You must have .NET 8 (or higher) installed
-* You must have Azure CLI (az) installed
-* You must have Azure Developer CLI (azd) installed
-* You must have npm installed
-* You must be logged into Azure CLI and Azure Developer CLI
-
-## Key concepts
-
-Write Azure apps in 5 minutes using simplified opinionated APIs and declarative resource provisioning.
+* You must have [.NET 8](https://dotnet.microsoft.com/download) (or higher) installed
+* You must have [Azure Developer CLI](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd) (azd) installed
+* You must be logged into Azure Developer CLI (`azd auth login`)
 
 ### Walkthrough
 
-#### Create Server Project
+#### Create a console application
 
 In a command line window type
+
 ```dotnetcli
 mkdir cmdemo
 cd cmdemo
-mkdir server
-cd server
-dotnet new web
+dotnet new console
 ```
 
-Add `Azure.Projects.*` packages
+#### Install Required Packages
+
 ```dotnetcli
+dotnet add package Azure.Projects --prerelease
 dotnet add package Azure.Projects.Provisioning --prerelease
 dotnet add package Azure.Projects.AI --prerelease
 ```
-#### Use Azure Developer CLI to provision Projects
 
-Open `Program.cs` file and add the following two lines of code to the top of the file
+#### Write, Provision, and Run the Application
+
+Open `Program.cs` file and change the code to the following
 ```csharp
```

Comment:
Please make all code samples snippets.  We generally enforce this for all libraries so that all sample code is at least compile-checked nightly.   No objection if you'd like to defer that work to get this out, but if so please file an issue to track.

Reply:
will do in a subsequent PR